doc/design/OrionHDFSSink.md

```diff
@@ -24,27 +24,37 @@ Independently of the data generator, NGSI context data is always [transformed](f
+    * `csv-row`: A CSV line is added for each notified context attribute. As `json-row`, this kind of line will always contain 8 fields:
+        * `recvTimeTs`: UTC timestamp expressed in miliseconds.
+        * `recvTime`: UTC timestamp in human-redable format ([ISO 6801](http://en.wikipedia.org/wiki/ISO_8601)).
+        * `entityId`: Notified entity identifier.
+        * `entityType`: Notified entity type.
+        * `attrName`: Notified attribute name.
+        * `attrType`: Notified attribute type.
+        * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be JSON object or JSON array.
+        * `attrMd`: In this case, the field does not contain the real metadata, but the name of the HDFS file storing such metadata. The reason to do this is the metadata may be an array of any length; each element within the array will be persisted as a single line in the metadata file. There will be a metadata file per each attribute under `/user/<hdfs_userame>/<fiware-service>/<fiware-servicePath>/<destination>_<attrName>_<attrType>/<destination>_<attrName>_<attrType>.txt`
+    * `csv-column`: A single CSV line is added for all the notified context attributes. This kind of line will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and another for the metadata, called `<attrName>_md_file` and containing the name of the HDFS file storing such metadata), plus an additional field about the reception time of the data (`recvTime`).
 
 ####Important notes regarding the persistence mode
-Please observe not always the same number of attributes is notified; this depends on the subscription made to the NGSI-like sender. This is not a problem for the `row` persistence mode, since fixed 8-fields Json documents are appended for each notified attribute. Nevertheless, the `column` mode may be affected by several Json documents of different lengths (in term of fields). Thus, the `column` mode is only recommended if your subscription is designed for always sending the same attributes, event if they were not updated since the last notification.
+Please observe not always the same number of attributes is notified; this depends on the subscription made to the NGSI-like sender. This is not a problem for the `row` persistence mode, since fixed 8-fields JSON/CSV documents are appended for each notified attribute. Nevertheless, the `column` mode may be affected by several JSON documents/CSV records of different lengths (in term of fields). Thus, the `column` mode is only recommended if your subscription is designed for always sending the same attributes, event if they were not updated since the last notification.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/62ac0c8bb28fb12ac9cd4fdb795ae2ca3631cdd5